spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -99,11 +107,24 @@ module.exports = function (provider, setup) {
       });
   });
 
-  it("Pause and Resume work", function (done) {
+  it("Closes socket and open new one on the same port", function (done) {
+    socket.listen('127.0.0.1', 9981, function () {
+      socket.close(function () {
+        socket.listen('127.0.0.1', 9981, function () {
+          expect(true).toEqual(true);
+          PromiseCompat.all([
+            socket.close(function () {}),
+            done()]);
+        });
+      });
+    });
+  });
+
+  it("Pauses and resumes", function (done) {
     socket.connect('www.google.com', 80, function () {
```

Comment:
Ah, damn github was hiding that line in the diff... makes sense. Thanks for pointing that out.